pkg/cache/task.go

```diff
@@ -213,12 +213,12 @@ func (task *Task) initialize() {
 
 	// task needs recovery means the task has already been
 	// scheduled by us with an allocation, instead of starting
-	// from New, directly set the task to Allocated.
+	// from New, directly set the task to Bound.
 	if utils.NeedRecovery(task.pod) {
 		task.allocationUUID = string(task.pod.UID)
 		task.nodeName = task.pod.Spec.NodeName
-		task.sm.SetState(TaskStates().Allocated)
-		log.Logger().Info("set task as Allocated",
+		task.sm.SetState(TaskStates().Bound)
```

Comment:
After analyzing the existing code, this seems to be a safe change. It is more sound to use Bound.

Reply:
See below